File: SoloLearn/Roadrunner/Roadrunner.rb
distance = gets.chomp.to_i
runnerSpeed = gets.chomp.to_i
coyoteSpeed = gets.chomp.to_i
runnerTime = distance / runnerSpeed
coyoteTime = (distance + 50) / coyoteSpeed
if runnerTime < coyoteTime
    puts "Meep Meep"
else
    puts "Yum"
end

File: SoloLearn/Roadrunner/Roadrunner.py
distance = int(input())
runnerSpeed = int(input())
coyoteSpeed = int(input())
runnerTime = distance / runnerSpeed
coyoteTime = (distance + 50) / coyoteSpeed
if runnerTime < coyoteTime:
    print("Meep Meep")
else:
    print("Yum")